landoapi/workers/landing_worker.py

```diff
@@ -86,11 +86,37 @@ def __init__(self, *args, **kwargs):
         self.last_job_finished = None
         self.refresh_enabled_repos()
 
-    def loop(self):
-        logger.debug(
-            f"{len(self.applicable_repos)} applicable repos: {self.applicable_repos}"
+    def check_landing_worker_warnings(self):
+        """Log messages that show various important statistics about the landing worker."""
+        TOO_MANY_ATTEMPTS_THRESHOLD = 10
+        # TODO: should this threshold be different for try landing worker?
+        QUEUE_SIZE_THRESHOLD = 20
+        ACTIVE_STATUSES = (
+            LandingJobStatus.SUBMITTED,
+            LandingJobStatus.DEFERRED,
+            LandingJobStatus.IN_PROGRESS,
+        )
```

Comment:
This tuple is a copy of the `applicable_statuses` variable in `job_queue_query`. Let's move them to a variable where they can be shared (maybe a class-level `LandingJobStatus.active_statuses` or something along those lines).